### PR TITLE
DIsplay toasts on beatmap/skin save

### PR DESCRIPTION
**Pull request by LukynkaCZE**
This is thing I have been missing in lazer for a long time. When mapping or editing skin I hit the save button/ctrl + s and I get no feedback from the game if it was saved or not. This is also feature in stable that is missing in lazer.

https://user-images.githubusercontent.com/48604271/179846239-7cf8eefc-94ab-4f61-87c4-a20c7c4ab02f.mp4

https://user-images.githubusercontent.com/48604271/179846245-de05fe8e-59ab-4a54-b86d-13286a9793bc.mp4

.

### CODE REVIEW

File: osu.Game/Localisation/ToastStrings.cs [hunk 1]
```diff
@@ -34,6 +34,26 @@ public static class ToastStrings
...
+
+        /// <summary>
+        /// "Beatmap Saved"
+        /// </summary>
+        public static LocalisableString EditorSaveBeatmap => new TranslatableString(getKey(@"beatmap_editor_save"), @"Beatmap Saved");
+
+        /// <summary>
+        /// "Skin Editor"
+        /// </summary>
+        public static LocalisableString SkinEditor => new TranslatableString(getKey(@"skin_editor"), @"Skin Editor");
```
Comment (peppy) [inline osu.Game/Localisation/ToastStrings.cs:50]: Use `SkinSettingsStrings.SkinLayoutEditor`

File: osu.Game/Localisation/ToastStrings.cs [hunk 2]
```diff
@@ -34,6 +34,26 @@ public static class ToastStrings
         /// </summary>
         public static LocalisableString RestartTrack => new TranslatableString(getKey(@"restart_track"), @"Restart track");
 
+        /// <summary>
+        /// "Beatmap Editor"
+        /// </summary>r
+        public static LocalisableString BeatmapEditor => new TranslatableString(getKey(@"beatmap_editor"), @"Beatmap Editor");
```
Comment (peppy) [inline osu.Game/Localisation/ToastStrings.cs:40]: Use `EditorSection.EditorSection`.

File: osu.Game/Localisation/ToastStrings.cs [hunk 3]
```diff
@@ -34,6 +34,26 @@ public static class ToastStrings
...
+
+        /// <summary>
+        /// "Beatmap Saved"
+        /// </summary>
+        public static LocalisableString EditorSaveBeatmap => new TranslatableString(getKey(@"beatmap_editor_save"), @"Beatmap Saved");
```
Comment (peppy) [inline osu.Game/Localisation/ToastStrings.cs:45]: I have no idea what's being done here.

Are you manually making these files? We generally use a rider tool to do this.

If you are manually doing it, you will need to pay very close attention to the format. The property name should match the key should match the string.

In this case, the string should be `Beatmap saved` (not title case) and the key should be `BeatmapSaved` / `beatmap_saved` to match.

Same applies to others.

**Review by peppy — CHANGES_REQUESTED** [review]

**Comment by LukynkaCZE** [comment]
I did all the requested changes 👍🏻  Also I wasn't aware there was tool to do the Localization stuff.. Well now I know and I'm gonna be using it from now on

**Review by peppy — DISMISSED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**